util/make_timeline.py

```diff
@@ -371,7 +371,7 @@ def main(args):
         if entry["line_type"] == "34":
             output_entry = '{position:.1f} "{targetable}"'.format(**entry)
```

Comment:
Are these targetable lines still ok?

Reply:
These don't include syncs ever, it just writes it in as `1000.0 "--targetable--"`. (When I get to the Typescript re-write, I may add the ability to sync these optionally, but it's not worth trying to add that to the Python version here.)